src/components/PasswordInput/component.tsx

```diff
@@ -3,20 +3,21 @@ import React, { useState, useCallback } from 'react';
 import { Testable, useBuildTestId } from '../../modules/test-ids';
 import { TextInput } from '../TextInput';
 import { Icon } from '../Icon';
+import { ValidationMessage } from '../internal/ValidationMessage';
 import { Styleless } from '../Styleless';
 
 export type Props = Omit<React.ComponentProps<typeof TextInput>, 'type' | 'left' | 'right'> &
   Testable & {
     isValid: boolean;
-    validationMessage: React.ReactNode;
+    validationMessages?: ValidationMessage[];
```

Comment:
You should be able to remove this line, since this prop is already coming from `React.ComponentProps<typeof TextInput>` above.